Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -303,7 +303,7 @@
 FEVTDEBUGHLTEventContent = cms.PSet(
     outputCommands = cms.untracked.vstring('drop *'),
     splitLevel = cms.untracked.int32(0),
-    eventAutoFlushCompressedSize=cms.untracked.int32(10*1024*1024)
+    eventAutoFlushCompressedSize=cms.untracked.int32(1*1024*1024)
```

Comment:
was this PR appropriately (re)based?
This change for 10\* to 1\* is undoing the version in 80X.